ui/pages/multichain-accounts/multichain-accounts-connect-page/index.scss

```diff
@@ -5,8 +5,4 @@
     height: 32px;
     width: 32px;
   }
-
-  &__tab {
-    flex: 1;
-  }
```

Comment:
We want all tabs to be the same width as the content so removing these customized styles